mesheryctl/pkg/utils/TestConfig.yaml

```diff
@@ -1,23 +1,23 @@
 contexts:
   local:
     endpoint: http://localhost:9081
-    token:
-      name: Default
-      location: auth.json
+    token: Default

```

Comment:
@aneeshnema I missed this before. How is the testconfig.yaml used? Do unit tests make use of this?

Reply:
Yes, the `TestViewCmd` unit test inside `channel_test.go` uses it.

Reply:
Wonderful. Unit tests in-action!